modules/openapi-generator/src/test/java/org/openapitools/codegen/TestUtils.java

```diff
@@ -173,6 +173,18 @@ public static void assertFileContains(Path path, String... lines) {
         }
     }
 
+    public static void assertFileContains(Path path, String line, int lineNumber) {
+        try {
+            List<String> lines = Files.readAllLines(path);
+            assertNotNull(lines);
+
+            String target = lines.get(lineNumber - 1);
+            assertTrue(target.contains(line), "File does not contain line [" + line + "] at lineNumber [" + lineNumber + "]");
```

Comment:
If we make this a:

```suggestion
            assertTrue(target.contains(line), "Expected "+ path + ":" + lineNumber + " to contain [" + line + "]");
```

Then links become clickable in the command line